saleor/graphql/schema.graphql

```diff
@@ -3125,14 +3110,29 @@ input PluginSortingInput {
 type PluginUpdate {
   errors: [Error!]!
   plugin: Plugin
-  extensionsErrors: [ExtensionsError!]!
+  pluginsErrors: [PluginsError!]!
```

Comment:
@d-wysocki Please open an issue in `saleor-dashboard` repo to update the field name there as well.

Reply:
Done, 
https://github.com/mirumee/saleor-dashboard/issues/472